src/psyclone/dynamo0p1.py

```diff
@@ -41,6 +41,7 @@
 from __future__ import absolute_import
 from psyclone.configuration import Config
 from psyclone.psyir.nodes import Loop, Literal, Reference, Array, Schedule
+from psyclone.psyir.symbols import DataSymbol, DataType
```

Comment:
DataType is already imported a couple of lines below.

Reply:
Oops. I didn't take much notice of gocean0.1 and dynamo0.1